.gen/dev.clab.yml.tpl

```diff
@@ -10,13 +10,4 @@ topology:
       binds:
         - "../{{ getenv "APPNAME" }}:/opt/{{ getenv "APPNAME" }}" # mount dir with agent code
         - "../logs/srl1:/var/log/srlinux" # expose srlinux logs to a dev machine
-        - "../{{ getenv "APPNAME" }}.yml:/etc/opt/srlinux/appmgr/{{ getenv "APPNAME" }}.yml" # put agent config file to appmgr directory
-    srl2:
-      binds:
-        - "../{{ getenv "APPNAME" }}:/opt/{{ getenv "APPNAME" }}" # mount dir with agent code
-        - "../logs/srl2:/var/log/srlinux" # expose srlinux logs to a dev machine
-        - "../{{ getenv "APPNAME" }}.yml:/etc/opt/srlinux/appmgr/{{ getenv "APPNAME" }}.yml" # put agent config file to appmgr directory
-  links:
-    - endpoints:
-        - "srl1:e1-1"
-        - "srl2:e1-1"
```

Comment:
let's not remove the second node from the template.
It is lightweight enough to keep the two, and it leaves a room for users to leverage the generated dev environment to kickstart their own agent development and it might require a second node running.

Reply:
> 

Perfect, I will add it back